mizar/obj/endpoint.py

```diff
@@ -364,3 +375,109 @@ def update_agent_substrate(self, ep, bouncer):
 
     def delete_agent_substrate(self, ep, bouncer):
         self.rpc.delete_agent_substrate_ep(ep, bouncer.ip)
+
+    def update_networkpolicy_per_endpoint(self, data):
+        if len(data["old"]) > 0:
+            self.delete_network_policy_ingress("no_except", data["old"]["ingress"]["cidr_table_no_except"])
+            self.delete_network_policy_ingress("with_except", data["old"]["ingress"]["cidr_table_with_except"])
```

Comment:
Cathy, let's discuss how we will combine these into batch. And shall we combine delete and update into batch.